src/olympia/editors/tests/test_views.py

```diff
@@ -350,6 +350,32 @@ def test_comment_logs(self):
         assert pq(r.content)('#log-listing tr td a').eq(1).text() == (
             'commented')
 
+    def test_review_url(self):
+        self.login_as_admin()
+        addon = addon_factory()
+        unlisted_version = version_factory(
+            addon=addon, channel=amo.RELEASE_CHANNEL_UNLISTED)
+
+        ActivityLog.create(
+            amo.LOG.APPROVE_VERSION, addon, addon.current_version,
+            user=self.get_user(), details={'comments': 'foo'},
+            created=self.days_ago(1))
```

Comment:
I forgot `ActivityLog.create` is a method that doesn't accept arbitrary keywords like `created` .... so you'll need to do
something like `al = ActivityLog.create(....)` then `al.update(created=self.days_ago(1))`